tests/errmsgs/tinteger_literals.nim

```diff
@@ -1,15 +1,14 @@
 discard """
-cmd: "nim check $file"
-errormsg: "number out of range: '300'u8'"
```

Comment:
Again, `errmsg` currently overrides `nimout`, so I got rid of both in this test and inlined the messages.